docker-compose.yml

```
@@ -0,0 +1,37 @@
+services:
+  db:
+    image: postgres:16
+    container_name: postgres_db
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: postgres
+    ports:
+      - 5432:5432
+    volumes: 
+      - postgres_data:/var/lib/postgresql/data
```

```suggestion
      - ./postgres-data:/var/lib/postgresql/data
```
Ne semble plus clair